requirements-dev.txt

```diff
@@ -18,3 +18,4 @@ scipy
 sympy
 wheel
 setuptools>=41.4.0
+neural-compressor
```

Comment:
Doesn't neural-compressor need to greater or equal than a specific version like v2.2?